### DISCUSSION TITLE
Composite Foreign Key relationships in the Diagram broken up

### DISCUSSION THREAD

**Discussion by hardysabs2** [dbeaver/dbeaver#22723] [General]
I have a PostgreSQL database that has a 3 composite FKs from one particular table to another.

I 'always used to' see these relationships in the diagram as a single line each with two fields highlighted in green in the tables on each side of the relationship.

Suddenly I am seeing these as 6 separate lines with only one field highlighted in green in the tables on each side of the relationship.

However if I drill down from the properties tab, the detail of these three composite keys appears to still be correct.

Anyone any ideas?  
Is there a diagram setting that maybe got defaulted during a DBeaver upgrade or something?

Thanks.

**Comment by LonwoLonwo** [comment]
Hello @hardysabs2 
Could you please provide DDL examples for reproducing?
